src/Symfony/Component/Serializer/NameConverter/NameConverterInterface.php

```diff
@@ -21,16 +21,20 @@
     /**
      * Converts a property name to its normalized value.
      *
-     * @param string $propertyName
+     * @param string|object $class
+     * @param string        $propertyName
+     *
      * @return string
      */
-    public function normalize($propertyName);
+    public function normalize($class, $propertyName);
```

Comment:
An easy way to limit the BC break is to inverse parameters (same for `denormalize`).